source/read/specify-documents-to-return.txt

```diff
@@ -0,0 +1,198 @@
+.. _kotlin-sync-specify-documents-to-return:
+
+===========================
+Specify Documents to Return
+===========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: read, paginate, pagination, order, code example
+
+Overview
+--------
+
+In this guide, you can learn how to specify which documents to return from a read
+operation by using the following methods:
+
+- ``limit()``: Specifies the maximum number of documents to return from a query.
+- ``sort()``: Specifies the sort order for the returned documents.
+- ``skip()``: Specifies the number of documents to skip before returning query results.
```

Comment:
I: [I don't think these should have periods](https://www.mongodb.com/docs/meta/style-guide/style/lists/#list-items)

```suggestion
- ``limit()``: Specifies the maximum number of documents to return from a query
- ``sort()``: Specifies the sort order for the returned documents
- ``skip()``: Specifies the number of documents to skip before returning query results
```